### DISCUSSION TITLE
Creating Defensibility for Watchman to drive growth, revenue, and industry leadership

### DISCUSSION THREAD

**Discussion by Sreenuraju** [moov-io/watchman#356] [Ideas]
@alrightjones         is there interest in  optimizing the defensibility of Watchman via testing, validation and benchmarking, with the objectives of:

1.   **Industry Leadership** -  positioning Moov as a national leader in KYC standards setting 
2.   **Best Practices**- engaging the DOJ and Federal Agencies (e.g. Treasury Dept, FinCen) in our benchmarking project to  contribute to Watchman's efficacy
3.  **Revenue**- generating revenue by selling a Watchman-Pro version that is validated to have acceptable levels of accuracy (i.e. precision/recall).  If possible, the Pro version would have user configurable settings to manage precision/recall and other statistical measures.    
4.   **Community**- engages the Moov community to contribute their efforts to optimizing the efficacy of Watchman

If there is interest, I would love to the Moov team's suggestions and help in creating a project design that addresses (in no particular order): taxonomy, record collection, relevance judgements, edge cases, evaluation measures, and business model.  

I appreciate being able to suggest this and believe this is a opportunity for Moov to take the mantle of leadership.

**Comment by adamdecaf** [comment]
Thanks for opening this discussion! I'd be very interested in better/public benchmarks for Watchman along with officially validating it for usage in more fields and agencies. Internally we've talked about a "Pro" version, so folks should stay tuned for that. 

We want to offer best of class service in our open-source projects and aim to share anything we can.

**Reply by Sreenuraju** [reply]
I would love to create a program that supports benchmarking that aligns with accepted methodologies on the judicial and regulatory policy setting side.  That's something that I know.      However, to design a benchmarking/validation program specific to Watchman and KYC,  there is so much I don't understand regarding the taxonomy, use cases, edge cases, and precisely WHAT and HOW Watchman works.  Would you be interested in setting up some time to discuss?

**Reply by adamdecaf** [reply]
@Sreenuraju Yea, can you setup a time on my calendar?
